src/s2wasm.h

```diff
@@ -1342,6 +1346,40 @@ class S2WasmBuilder {
     }
   }
 
+  // Converts invoke wrapper names generated by LLVM backend to real invoke
```

Comment:
I don't know how I can make it more general. You mean, it should not sound like a LLVM specific thing? But what this function does is basically converting LLVM IR type names into wasm type names in function signatures, so I don't know how it can be independent of LLVM.